bundles/binding/org.openhab.binding.zibase/src/main/java/org/openhab/binding/zibase/internal/ZibaseGenericBindingProvider.java

```diff
@@ -49,6 +49,11 @@
     static final HashMap<String, Vector<String>> itemIdNameMap = new HashMap<String, Vector<String>>();
 
     /**
+     * Map that allows to get registered item by name
+     */
+    static final HashMap<String, Item> itemMap = new HashMap<String, Item>();
```

Comment:
Hmmm, you're right ! After re-reading my code I can remove this items hashmap. I used it for some debugging. I will remove in the next couples of days.